rest/model/buildlogger.go

```diff
@@ -0,0 +1,106 @@
+package model
+
+import (
+	dbmodel "github.com/evergreen-ci/cedar/model"
+	"github.com/pkg/errors"
+)
+
+// APILog describes metadata for a buildlogger log.
+type APILog struct {
+	ID          APIString          `json:"id,omitempty"`
+	Info        APILogInfo         `json:"info,omitempty"`
+	CreatedAt   APITime            `json:"created_at"`
+	CompletedAt APITime            `json:"completed_at"`
```

Comment:
should we compute a duration (float seconds) for people?

Reply:
is that something useful? also, can't that just be calculated in the UI?